src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    String draw();
```

Comment:
you should not return any strings here, just print info
![image](https://github.com/mate-academy/jv-oop-advanced/assets/73041091/66e88f51-f7ff-448d-a981-a273b385d823)
```suggestion
    void draw();
```